doc/contributing/docs/class_template.rst

```diff
@@ -0,0 +1,43 @@
+..  class:: index_object
```

Comment:
Please put this file into a dedicated `./_include` directory, and add a template like `./**/_include/*` to the exclusion list in conf.py.

Reply:
Same with the other included file.

Reply:
sure, thanks